llama-index-legacy/llama_index/legacy/vector_stores/pinecone_utils.py

```diff
@@ -28,3 +28,13 @@ def _is_pinecone_v3() -> bool:
     ):  # Will not work with .dev versions, e.g. "3.0.0.dev8"
         return True
     return False
+
+def _is_serverless(index_name: str) -> bool:
```

Comment:
I think you probably meant to update the utils in `llama-index-integrations/vector_stores/llama-index-vector-stores-pinceone`